### DISCUSSION TITLE
How to return results where managed property doesn't contain specific special characters

### DISCUSSION THREAD

**Discussion by JonoSuave** [microsoft-search/pnp-modern-search#2029] [Q&A]
I have some search results where the Title property value begin with ~$. I'd like to exclude those documents. Any approaches you'd suggest? I tried -Title:`~$ but no luck.
<img width="346" alt="Screen Shot 2022-04-19 at 5 49 14 PM" src="https://user-images.githubusercontent.com/22969971/164119294-ac5eba35-8fe5-46f6-b854-e39f3d218cfb.png">

**Comment by wobba** [comment]
See https://docs.microsoft.com/en-us/sharepoint/search/manage-the-search-schema about complete matching. Special characters are typically tokenized away, so you cannot search on them by default.
